tests/storage/clone.go

```diff
@@ -6,6 +6,9 @@ import (
 	"strings"
```

Comment:
Not sure clone tests should be in "storage" package.  The controller is not in "storage" subpackage